docs/Tutorials/CCE.md

```diff
@@ -199,6 +202,14 @@ supports \f$\ell\in[4, 32]\f$.
 \snippet Test_Spherepack.cpp spectre_cce_grid_point_locations
 </details>
 
+Alternatively, if your code can read in grid points from a text file, you can
```

Comment:
text file or h5? Because the `-o` below is with a `.h5`. You can do this when you squash :)